packages/client/bin/cli.ts

```diff
@@ -207,7 +207,8 @@ const args: ClientOpts = yargs(hideBin(process.argv))
   .option('rpcDebug', {
     describe:
       'Additionally log complete RPC calls filtered by name (prefix), e.g.: "eth,engine_getPayload" (use "all" for all methods).',
-    default: '',
```

Comment:
Using a default value can be more intuitive than removing it and instead having to hand `undefined` as a value.